src/commands/protocols/publish/receiver/v1.0.0/v1-0-0-handle-store-request-command.js

```diff
@@ -46,6 +46,13 @@ class HandleStoreRequestCommand extends HandleProtocolMessageCommand {
             tokenId,
         );
         const stateIndex = assertionIds.length - 1;
+
+        if (assertionId !== assertionIds[stateIndex]) {
```

Comment:
I think we already do this in validation command

Reply:
We don't